docs/tutorials/install_and_admin/geonode_update/26_to_27/index.txt

```diff
@@ -135,8 +135,8 @@ Upgrade Development Environment
             64
             65  GEOSERVER_PUBLIC_LOCATION = os.getenv(                                                                                  49  GEOSERVER_PUBLIC_LOCATION = os.getenv(
             ------------------------------------------------------------------------
-            66  #    'GEOSERVER_PUBLIC_LOCATION', '{}geoserver/'.format(SITEURL)                                                    <>
-            67      'GEOSERVER_LOCATION', 'http://localhost:8080/geoserver/'                                                            50      'GEOSERVER_PUBLIC_LOCATION', 'http://localhost:8080/geoserver/'
+            66  #    'GEOSERVER_PUBLIC_LOCATION', '{}gs/'.format(SITEURL)                                                    <>
+            67      'GEOSERVER_LOCATION', 'http://localhost:8080/geoserver/'                                                            50      'GEOSERVER_PUBLIC_LOCATION', 'http://localhost:8080/gs/'
```

Comment:
This URL is not correct, it should be either:

http://localhost:8080/geoserver/

or 

http://localhost:8000/gs/

not a mix of the two above.

Reply:
@afabiani right, sorry I did not realize of the sphinx diff visualization. If I understand correctly you mean:

```
65  GEOSERVER_PUBLIC_LOCATION = os.getenv(                                                                                  49  GEOSERVER_PUBLIC_LOCATION = os.getenv(
            ------------------------------------------------------------------------
            66  #    'GEOSERVER_PUBLIC_LOCATION', '{}geoserver/'.format(SITEURL)                                                    <>
            67      'GEOSERVER_LOCATION', 'http://localhost:8080/geoserver/'                                                            50      'GEOSERVER_PUBLIC_LOCATION', 'http://localhost:8080/gs/'
            68  )

```

right?

Reply:
Right. You should also make sure this not happens on other lines below.